transactions/stage-dependency-contract/2024/aug-27/arguments.json

```diff
@@ -0,0 +1 @@
02020202020202f2f2060636865636b446570656e64656e6369657360206973206f7468657277697365206e6f742063616c6c61626c652066726f6d20746865206f7574736964650a202020202020202061636365737328616c6c292066756e2074657374436865636b446570656e64656e63696573285f20646570656e64656e636965734164647265737365733a205b416464726573735d2c205f20646570656e64656e636965734e616d65733a205b537472696e675d2c205f20617574686f72697a6572733a205b416464726573735d29207b0a20202020202020202020202072657475726e20446570656e64656e637941756469742e636865636b446570656e64656e6369657328646570656e64656e636965734164647265737365732c20646570656e64656e636965734e616d65732c20617574686f72697a657273290a20202020202020207d0a202020207d0a0a2020202061636365737328616c6c292073747275637420446570656e64656e6379207b0a202020202020202061636365737328616c6c29206c657420616464726573733a20416464726573730a202020202020202061636365737328616c6c29206c6574206e616d653a20537472696e670a0a2020202020202020696e697428616464726573733a20416464726573732c206e616d653a20537472696e6729207b0a20202020202020202020202073656c662e61646472657373203d20616464726573730a20202020202020202020202073656c662e6e616d65203d206e616d650a20202020202020207d0a0a202020202020202061636365737328616c6c292066756e20746f537472696e6728293a20537472696e67207b0a2020202020202020202020207661722061646472657373537472696e67203d2073656c662e616464726573732e746f537472696e6728290a2020202020202020202020202f2f2072656d6f7665203078207072656669780a20202020202020202020202061646472657373537472696e67203d2061646472657373537472696e672e736c6963652866726f6d3a20322c207570546f3a2061646472657373537472696e672e6c656e677468290a20202020202020202020202072657475726e2022412e222e636f6e6361742861646472657373537472696e67292e636f6e63617428222e22292e636f6e6361742873656c662e6e616d65290a20202020202020207d0a202020207d0a0a202020202f2f205468652061646d696e207265736f7572636520697320736176656420746f207468652073746f7261676520736f2074686174207468652061646d696e2063616e206265206163636573736564206279207468652073657276696365206163636f756e740a202020202f2f2054686520606578636c75646564416464726573736573602077696c6c20626520746865206164647265737365732077697468207468652073797374656d20636f6e7472616374732e0a20202020696e6974286578636c756465644164647265737365733a205b416464726573735d29207b0a202020202020202073656c662e6578636c75646564416464726573736573203d207b7d0a202020202020202073656c662e70616e69634f6e556e737461676564203d2066616c73650a0a202020202020202073656c662e41646d696e6973747261746f7253746f7261676550617468203d202f73746f726167652f666c6f77446570656e64656e6379417564697441646d696e0a0a2020202020202020666f72206164647265737320696e206578636c75646564416464726573736573207b0a20202020202020202020202073656c662e6578636c756465644164647265737365735b616464726573735d203d20747275650a20202020202020207d0a0a20202020202020206c65742061646d696e203c2d206372656174652041646d696e6973747261746f7228290a202020202020202073656c662e6163636f756e742e73746f726167652e73617665283c2d61646d696e2c20746f3a2073656c662e41646d696e6973747261746f7253746f7261676550617468290a202020207d0a7d0a"}]
```

Comment:
@janezpodhostnik - arg format look ok?

Reply:
or should it be in ascii format?

Reply:
It should be ascii. @sisyphusSmiling can you confirm?

Reply:
Yes this looks like correct formatting

Reply:
If I look at this tx on testnet it has ascii formatting not hex https://testnet.flowdiver.io/tx/30d59d1e256a18e487612b0371ea4b3f3e4d0623e0195c0cef4d08a88f5d2bcd?tab=script.